ui/nuxeo-user-group-management/nuxeo-create-group.js

```diff
@@ -133,7 +133,11 @@ import '../nuxeo-button-styles.js';
                   <div class="email">
                     <span>[[item.email]]</span>
                   </div>
-                  <paper-icon-button icon="nuxeo:remove" title="remove" on-click="_remove"></paper-icon-button>
+                  <paper-icon-button
+                    icon="nuxeo:remove"
+                    title$="[[i18n('command.remove')]]"
+                    on-click="_remove"
```

Comment:
no `aria-labelledby` or `aria-label` here?

Reply:
it isn't needed because it has a title already: https://dequeuniversity.com/rules/axe/4.1/button-name

Reply:
Awsome. And it works even for custom elements, right?

Reply:
Yes, I just tested with Apple Voice Over tool and it works. It doesn't matter if it is a custom element since in this case `paper-button` sets a `role=button` which is picked up by screen readers.